active/0047-node-support.md

```diff
@@ -0,0 +1,28 @@
+- Start Date: 2016-04-04
+- RFC PR: #47
+
+# Summary
+
+One of the nice things about Node.js being our server environment is that we are not beholden to the vast installed base of browsers and can instead move our community by fiat. However, we've never explicitly communicated what our plans for support will be and our current pattern has grown organically. We need to define these guidelines so that larger organizations are able to plan for future migrations.
+
+# Motivation
+
+We've avoided adding libraries like Babel into our ember-cli applications in order to maintain debuggability and reduce magic. This leaves us yearning for the features, flexibility, and syntactic sugar of more-recent V8 releases. In order to not be disruptive we have continued to support the 0.10 branch which has tied us to an ancient V8 engine.
+
+# Proposal
+
+We track the [existing LTS releases for Node.js](https://github.com/nodejs/LTS#lts_schedule), but only provide support for the active LTS releases, or the stable versions which will become LTS releases. Once an LTS is moved into maintenance mode we will drop support for that version. Versions of Ember CLI whose support crosses one of the Node.js boundaries will be supported per the ember-cli release support policy which is currently unspecified, though described by RFC #46.
```

Comment:
Can you spell this sentance out a bit more? Assuming this policy has been in place from the beginning, what would we be supporting today? What will we be supporting after 2017-04-01, etc?